tier4_deprecated_api_adapter/src/manual_control.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MANUAL_CONTROL_HPP_
+#define MANUAL_CONTROL_HPP_
```

Comment:
Perhaps / Personally `TIER4_DEPRECATED_API_ADAPTER__MANUAL_CONTROL_HPP_` might be a safe choice to prevent name duplication in future development.

```suggestion
#ifndef TIER4_DEPRECATED_API_ADAPTER__MANUAL_CONTROL_HPP_
#define TIER4_DEPRECATED_API_ADAPTER__MANUAL_CONTROL_HPP_
```

Reply:
This file is a private header, so it does not need a package name. This is automated by pre-commit. If the file is placed in the include directory, the package name is added.

Reply:
I understand, thank you for your detailed explanation!